config/locales/server.en.yml

```diff
@@ -1686,6 +1686,7 @@ en:
     default_categories_watching_first_post: "List of categories in which first post in each new topic will be watched by default."
 
     retain_web_hook_events_period_days: "Number of days to retain web hook event records."
+    retry_web_hook_events: "Retry web hook events if failed."
```

Comment:
I think we need to improve the description here to include the number of retries and how long it is between each retry.